app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -92,14 +92,30 @@ do (ko, $=jQuery) ->
 
   ko.bindingHandlers.echoform =
     init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
+      @email = ""
```

Comment:
I don't think we need to do all of this. In Chrome, `edsc.page.account.email()` while on /data/configure gives me the URS email address of the logged in user. I think accessing the email address already stored on the client would be better.